stock_picking_report_cmr/views/report_cmr.xml

```diff
@@ -0,0 +1,1225 @@
+                                                        <tr>
+                                                            <td
+                                                                style="font-size: 1.5em"
+                                                            >
+                                                                <strong>10</strong>
+                                                            </td>
+                                                            <td class="pl-1">
+                                                                Numero di statistica
+                                                                <br />
+                                                                N.º statistique
+                                                                <br />
+                                                                Statistical number
+                                                            </td>
+                                                        </tr>
+                                                    </table>
+                                                </td>
+                                                <td style="border-left: 2px solid">
+                                                    <table class="table-inner">
+                                                        <tr>
+                                                            <td
+                                                                style="font-size: 1.5em"
+                                                            >
+                                                                <strong>11</strong>
+                                                            </td>
+                                                            <td class="pl-1">
+                                                                Peso lordo kg.
+                                                                <br />
+                                                                Poids brut Kg.
+                                                                <br />
+                                                                Gross weight in Kg
+                                                                <t
+                                                                    t-if="o.shipping_weight"
+                                                                >
+                                                                    <div
+                                                                        class="text-bigger text-dark mt-1 ml-2"
+                                                                    >
+                                                                        <span
+                                                                            t-field="o.shipping_weight"
+                                                                        />
```

Comment:
@ntsirintanis make it `o.weight` rather - I don't see `shipping_weight` being filled, on second viewing.

Reply:
shipping_weight is basically computed from package weight. Have you checked if the weight field in packages if filled?